evap/results/templates/results_course_detail.html

```diff
@@ -19,8 +19,6 @@
 
     {% if course.state != 'published' %}
         <div class="alert alert-warning">{% trans 'This is a preview. The course is still in the process of evaluation.' %}</div>
-    {% elif not course.has_enough_voters_to_publish_grades %}
-        <div class="alert alert-warning"><span class="far fa-eye-slash"></span> {% blocktrans %}The results of this course have not been published because it didn't get enough votes.{% endblocktrans %}</div>
```

Comment:
why did you remove this?

Reply:
before, staff could see the average grade even if it wasn't published so we needed a warning to tell them that it's not public. now they can't see the average grade either and the gray distribution bar is sufficient explanation.